xml/ha_storage_protection.xml

```diff
@@ -664,7 +664,18 @@ Timeout (msgwait)  : 10
    <procedure xml:id="pro-ha-storage-protect-sbd-config">
    <title>Editing the SBD configuration file</title>
     <step>
-     <para>Open the file <filename>/etc/sysconfig/sbd</filename>.</para>
+     <para>Open the file <filename>/etc/sysconfig/sbd</filename> and use
+      the following entries:</para>
+     <screen>SBD_PACEMAKER=yes
+SBD_STARTMODE=always
+SBD_DELAY_START=no
+SBD_WATCHDOG_DEV=/dev/watchdog
+SBD_WATCHDOG_TIMEOUT=5</screen>
+      <para>
+       The <varname>SBD_DEVICE</varname> entry is not needed as no shared
+       disk is used. When this parameter is missing, the <systemitem>sbd</systemitem>
```

Comment:
This text applies well for the case of diskless SBD. But we are in the context of SBD with shared disk here, right?